test/cypress/integration/widgets/cluster_status_spec.ts

```diff
@@ -78,9 +78,11 @@ describe('Cluster Status widget', () => {
             cy.get('.popup').should('not.exist');
         };
 
-        cy.interceptSp('GET', 'cluster-status?summary=false', { fixture: 'cluster_status/fail_with_services.json' }).as(
-            'clusterStatusFailWithServices'
-        );
+        cy.interceptSp(
+            'GET',
+            { path: '/cluster-status?summary=false' },
```

Comment:
Can we have here:
```suggestion
            { pathname: '/cluster-status', query: { summary: 'false' } }
```
?

Reply:
Semantically it's not the same. In the original form the assert is stronger. It checks that there is `summary` parameter and no other parameter, as the whole path must match exactly. In the suggested form it checks if there is `summary` parameter, but there still may be other parameters and the request would still match. I decided not to relax assertions in such cases. The same applies to other occurrences as well.

Reply:
OK, make sense. In that case I'll resolve my other comments related to that thing.